fermenter-mda/pom.xml

```diff
@@ -174,6 +166,9 @@
                         <phase>process-classes</phase>
                     </execution>
                 </executions>
```

Comment:
I: annoyingly required with the latest version of maven plugin tools.  Seems likely to be rolled back in a future version, but here for now.